### ISSUE TITLE
mobile app stops working on all devices

### ISSUE THREAD

**Issue by 3s-hosting** [bitwarden/mobile#527]
a while ago all our apps on mobile phone stopped syncing. as the app still was logged in it was not detected immediately by us so I cannot tell what exactly happend or when exactly the apps stopped working. All mobile phones are affected: Android, IOS. It doesn't matter if local network or from outside.
The vault can not be opened on these devices too. Loading the page in Chrome or Firefox or trying to login in the app takes AGES till timeout.
Today I updated to bitwarden server 1.30.4. The problem still exists.

Even the replacement of the company firewall didn't help.
The firewall on the linux machine ist deactivated and flushed except the needed rules to allow bitwarden to send mails (or other traffic) out.

The only helpful message I get on the server from docker logs -ft bitwarden-identity is:

2019-06-10T20:37:10.529481206Z info: Microsoft.AspNetCore.Hosting.Internal.WebHost[1]
2019-06-10T20:37:10.529527906Z       Request starting HTTP/1.0 POST http://ourURL/connect/token application/x-www-form-urlencoded 274
2019-06-10T20:37:10.529889206Z info: IdentityServer4.Hosting.IdentityServerMiddleware[0]
2019-06-10T20:37:10.529914006Z       Invoking IdentityServer endpoint: IdentityServer4.Endpoints.TokenEndpoint for /connect/token
2019-06-10T20:37:10.661650506Z warn: Microsoft.AspNetCore.Identity.UserManager[0]
2019-06-10T20:37:10.661697006Z       Invalid password for user some_cryptic_userid.
2019-06-10T20:37:12.688005006Z info: IdentityServer4.Validation.TokenRequestValidator[0]
2019-06-10T20:37:12.688050706Z       User authentication failed: invalid_username_or_password, request details: {
2019-06-10T20:37:12.688058906Z         "ClientId": "mobile",
2019-06-10T20:37:12.688072006Z         "GrantType": "password",
2019-06-10T20:37:12.688078506Z         "Scopes": "api offline_access",
2019-06-10T20:37:12.688092006Z         "UserName": "some username",
2019-06-10T20:37:12.688098506Z         "Raw": {
2019-06-10T20:37:12.688104006Z           "grant_type": "password",
2019-06-10T20:37:12.688109706Z           "username": "some username",
2019-06-10T20:37:12.688115306Z           "password": "***REDACTED***",
2019-06-10T20:37:12.688120706Z           "scope": "api offline_access",
2019-06-10T20:37:12.688126206Z           "client_id": "mobile",
2019-06-10T20:37:12.688131706Z           "DeviceType": "Android",
2019-06-10T20:37:12.688137106Z           "DeviceIdentifier": "some cryptic Identifier",
2019-06-10T20:37:12.688142606Z           "DeviceName": "some device name",
2019-06-10T20:37:12.688190106Z           "DevicePushToken": ""
2019-06-10T20:37:12.688198706Z         }
2019-06-10T20:37:12.688204106Z       }
2019-06-10T20:37:12.695283406Z info: Microsoft.AspNetCore.Hosting.Internal.WebHost[2]
2019-06-10T20:37:12.695330806Z       Request finished in 2165.8837ms 400 application/json; charset=UTF-8

everything else works fine. Win10-App, Web-Vault
certificate is not self signed and used by MANY of our services, no mobile phone has problems with this so I doubt that there could be a problem.

I'm not an expert on docker, am glad to have it running, so I'm out of ideas for further debugging, sorry about that.
If you need further information please let me know how to get them and I will try my best.

Best regards

ps: please excuse my horrible english ^^

**Comment by kspearrin** [comment]
There is a known issue where push notifications stop working on self-hosted servers after approx 48 hours. There is a fix coming in the next server version. For now, the workaround is to restart the server.

Ref: https://github.com/bitwarden/server/issues/501

**Comment by walkerprog** [comment]
sadly restart doesn't help.
The server was restarted about 30 minutes ago. All mobile apps still completely locked out, no chance to log in.

btw: passwords are definitely correct, can login with same username and password combination everywhere else, except in mobile app

**Comment by kspearrin** [comment]
Sounds like a different problem then. The referenced bug doesn't have anything to do with not being able to log in. What is the status of `docker ps`? This also doesn't sound like a mobile app problem if you cannot log in and access data from any type of device.

**Comment by walkerprog** [comment]
output of docker ps:

[root@THESERVER ~]# docker ps
CONTAINER ID        IMAGE                            COMMAND             CREATED             STATUS              PORTS                                                     NAMES
a2079733d15b        bitwarden/nginx:1.30.1           "/entrypoint.sh"    42 minutes ago      Up 42 minutes       80/tcp, SOMEIP:80->8080/tcp, SAMEIP:443->8443/tcp   bitwarden-nginx
0be7f73adb57        bitwarden/admin:1.30.1           "/entrypoint.sh"    42 minutes ago      Up 42 minutes       5000/tcp                                                  bitwarden-admin
42af17ef1813        bitwarden/api:1.30.1             "/entrypoint.sh"    42 minutes ago      Up 42 minutes       5000/tcp                                                  bitwarden-api
7378a10c3187        bitwarden/attachments:1.30.1     "/entrypoint.sh"    42 minutes ago      Up 42 minutes       5000/tcp                                                  bitwarden-attachments
84e60572ea92        bitwarden/mssql:1.30.1           "/entrypoint.sh"    42 minutes ago      Up 42 minutes       1433/tcp                                                  bitwarden-mssql
61f44584f2d9        bitwarden/icons:1.30.1           "/entrypoint.sh"    42 minutes ago      Up 42 minutes       5000/tcp                                                  bitwarden-icons
b4a9b5e00c78        bitwarden/web:2.10.0             "/entrypoint.sh"    42 minutes ago      Up 42 minutes       5000/tcp                                                  bitwarden-web
87e09c310552        bitwarden/identity:1.30.1        "/entrypoint.sh"    42 minutes ago      Up 42 minutes       5000/tcp                                                  bitwarden-identity
0067559bc909        bitwarden/notifications:1.30.1   "/entrypoint.sh"    42 minutes ago      Up 42 minutes       5000/tcp                                                  bitwarden-notifications


I don't really understand "1.30.1"

Restart AND Update wrote "1.30.4: Pulling from bitwarden/setup"

basically I CAN login from ANY device on ANY location (local net or elsewhere on the world) per web or Windows App. While my notebook with Windows app logs in an syncs our mobile phones will NOT
